saleor/graphql/shipping/types.py

```diff
@@ -284,6 +283,9 @@ class ShippingMethod(ChannelContextType):
     price = graphene.Field(
         Money, required=True, description="The price of selected shipping method."
     )
+    maximum_order_price = graphene.Field(
+        Money, description="Maximal order price for this shipping method."
```

Comment:
I believe this is something mine. Shouldn't be `Maximum`?

Reply:
Fixed